src/main/java/com/google/devtools/build/lib/bazel/commands/ModCommand.java

```diff
@@ -494,4 +510,38 @@ private static BlazeCommandResult createFailureResult(String message, Code detai
                 .setMessage(message)
                 .build()));
   }
+
+  public static void dumpRepoMapping(BazelDepGraphValue depGraphValue, Writer writer) {
+    try (JsonWriter jsonWriter = new JsonWriter(writer)) {
+      jsonWriter.beginObject();
```

Comment:
overall looks good! just one minor thing -- IMO JsonWriter is a bit too low-level for this; using JsonObject, JsonArray etc would be much more readable (and I doubt memory usage of `bazel mod dump_repo_mapping` is of any concern). See [JsonOutputFormatter](https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/bzlmod/modcommand/JsonOutputFormatter.java;l=58) for an example.